docs/concepts/admission-webhook.md

```diff
@@ -0,0 +1,44 @@
+# Admission-webhook
+
+Lokomotive has an admission-webhook which runs in it's own namespace `lokomotive`. This document explains how the webhook works adding more security to your cluster. 
+
+## Additional Security
+
+An admission controller is a piece of code that intercepts requests to the Kubernetes API server prior to persistence of the object, but after the request is authenticated and authorized. There are two special controllers: `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook`.
+
+When you create a pod, if you do not specify a service account, it is automatically assigned the `default` service account in the same namespace. If you get the raw json or yaml for a pod you have created (for example, `kubectl get pods/<podname> -o yaml`), you can see the `spec.serviceAccountName` field has been automatically set. For more information see [kubernetes docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/). 
+
+Not every pod needs the ability to utilize the API from within itself. For example, if there is a web application for voting that just counts votes and sends this information to an external data base, there is no need for the app to contact the API server, and therefore a mounted token is unnecessary. An attacker that finds a privileged token on a pod can compromise the cluster, so we need to reduce the attack surface as much as possible
+
+To avoid manually disabling automounting of default service account, we have a webhook server that patches your service account whenever you either apply any [lokomotive component](./components.md) or create a new namespace. To see how it works follow below steps from your command line.
+
+```bash
+# create a namespace
+$ kubectl create ns foo
+
+# get default service for foo namespace
+$ kubectl get sa default -o yaml -n foo
+```
+
+After following the above steps you can see that `automountServiceAccountToken` is set to false.
+
+## What does not work
+
+Currently, on applying cluster `default` service account for `default` namespace is not patched. If you want it patched too, delete the default service account after your cluster is created.
```

Comment:
Then this should be explained in subsection "'default' ServiceAccount in 'default' Namespace".